### DISCUSSION TITLE
Welcome to lwshell Discussions!

### DISCUSSION THREAD

**Discussion by MaJerle** [MaJerle/lwshell#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by giovani-luigi** [comment]
Hey!
I am Giovani Luigi, (yep with 1 'n' only  😄), I work with Embedded Systems overall. 
From tiny MCUs up to systems running on Win10 IoT.
My main skills/technologies are C# and .NET but recently I am getting some special feeling towards C++
I am from Brazil and work with a company in Canada. I develop GPS-based solutions for precision agriculture.
Currently, I'm trying to divide my time between my main two passions: microcontrollers and C#.
I've just posted a new discussion [Link to discussion](https://github.com/MaJerle/lwshell/discussions/2#discussion-3325253)
Hopefully, I will find some time to contribute to this community.
Regards, 
Giovani Luigi
